jobs/build/tarball-sources/Jenkinsfile

```diff
@@ -0,0 +1,148 @@
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: "ocp-build@rcm-guest.app.eng.bos.redhat.com:/mnt/redhat/staging-cds/"
+                    ],
+                    [
+                        name: 'MAIL_LIST_SUCCESS',
+                        description: 'Success Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'release-engineering@redhat.com',
+                        ].join(',')
+                    ],
+                    [
+                        name: 'MAIL_LIST_FAILURE',
+                        description: 'Failure Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'aos-art-automation+failed-release@redhat.com'
+                        ].join(',')
+                    ],
+                    commonlib.mockParam(),
+                ]
+            ],
+        ]
+    )
+
+    commonlib.checkMock()
+    buildlib.initialize()
+    buildlib.cleanWorkdir(env.WORKSPACE)
+
+    try {
+        sshagent(['openshift-bot']) {
+            advisory = params.ADVISORY ? Integer.parseInt(params.ADVISORY.toString()) : 0
+            if (advisory == 0) {
+                error "Need to provide an ADVISORY ID"
+            }
+            def components = params.COMPONENTS.replaceAll(',', ' ').split(' ')
+            def filename = "tarball-source-output.txt"
+            def outdir = "/mnt/nfs/home/jenkins/container-sources/"
+            def rcm_guest = params.RCM_GUEST.toString()
+            currentBuild.description += "tarball: ${components}"
+
+            stage("elliott tarball-source") {
+                component_args = ""
+                components.each {
+                    component_args += "--component ${it}"
+                }
+                buildlib.elliott """
+                    tarball-sources create
+                    --out-dir=${outdir}
+                    --force
+                    ${component_args}
+                    ${advisory} > ${filename}
+                """
+            }
+
+            stage("rsync") {
+                cmd = "rsync -avz --chmod=go+rX ${outdir} ${rcm_guest}"
+                commonlib.shell(
+                    script: cmd,
+                    returnAll: true
+                )
+            }
+
+            stage("file ticket to RCM") {
+                // use awk to print out the sigle line cotains ${advisory}
+                // sth like this:
+                // RHEL-7-OSE-4.2/${advisory}/release/logging-fluentd-container-v4.2.26-202003230335.tar.gz 
+                cmd = """awk '/${advisory}/ { save=\$0 }END{ print save }' ${filename}"""
+                verify = commonlib.shell(
+                        script: cmd,
+                        returnStdout: true
+                )
+
```

Comment:
Think a version indicator in the job description would be very nice. Maybe it can be prepended to the components? Could parse `${filename}` to obtain at least e.g. `4.2`...